docs/guides/dev_guide.rst

```diff
@@ -310,7 +317,7 @@ Layout
     fixture that exposes the utility to the tests. Modules in this directory
     will be auto loaded.
 
-  	* ``tests/`` Unit tests for utils
+    * ``tests/`` Unit tests for utils
```

Comment:
**REQUIRED:** This also is at the right level

Reply:
I am not sure about this one. It is already one level from the previous indent. I replaced 1 tab with 2 spaces here too.

Reply:
As we talked on Gitter, that should be ok, right?
Are you ok with the text changes too? Thanks in advance!